### PR TITLE
Chap5

### PR DESCRIPTION
**Pull request by huutung06021992**
chap5

### CODE REVIEW

**Comment by huutung06021992** [comment]
ready

**Comment by ghost** [comment]
commented
bạn gộp commit lại nhé

**Comment by huutung06021992** [comment]
ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,12 @@
 module ApplicationHelper
+    # Returns the full title on a per-page basis.
+  def full_title(page_title = "")
```
Comment (thangtx88) [inline app/helpers/application_helper.rb]: bỏ ()

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,12 @@
...
+    base_title = "Ruby on Rails Tutorial Sample App"
```
Comment (thangtx88) [inline app/helpers/application_helper.rb]: I18n

File: app/views/layouts/_footer.html.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+  <small>
+    The <a href="http://www.railstutorial.org/"><%= I18n.t("tutorial") %></a>
+    by <a href="http://www.michaelhartl.com/"><%= I18n.t("name") %></a>
+  </small>
+  <nav>
+    <ul>
+      <li><%= link_to I18n.t("about"),  "#" %></li>
+      <li><%= link_to I18n.t("contact"), "#" %></li>
+      <li><a href="http://news.railstutorial.org/">News</a></li>
```
Comment (thangtx88) [inline app/views/layouts/_footer.html.erb]: I18n

**Comment by thangtx88** [comment]
commented

**Comment by huutung06021992** [comment]
ready

**Comment by thangtx88** [comment]
Merged

**Pull request merged**